src/test/java/stirling/software/SPDF/controller/api/RotationControllerTest.java

```diff
@@ -0,0 +1,79 @@
+package stirling.software.SPDF.controller.api;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+
+import org.apache.pdfbox.pdmodel.PDPage;
+import java.io.IOException;
+import org.apache.pdfbox.pdmodel.PDPageTree;
+import org.junit.jupiter.api.BeforeEach;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import org.springframework.http.ResponseEntity;
+import org.springframework.mock.web.MockMultipartFile;
+import stirling.software.SPDF.service.CustomPDDocumentFactory;
+import stirling.software.SPDF.model.api.general.RotatePDFRequest;
+
+public class RotationControllerTest {
+
+    @Mock private CustomPDDocumentFactory pdfDocumentFactory;
+
+    @InjectMocks private RotationController rotationController;
+
+    @BeforeEach
+    public void setUp() {
```

Comment:
If you add `@ExtendWith(MockitoExtension.class)` on the class level you can remove this setup method